packages/smooth_app/lib/themes/smooth_theme.dart

```diff
@@ -47,8 +47,9 @@ class SmoothTheme {
 
   static MaterialColor getMaterialColor(
     final ThemeProvider themeProvider,
```

Comment:
You can remove the ThemeProvider here and get it inside `getMaterialColorImpl` since you already pass the context

Reply:
did the same, but some unit tests are failing Idk why? rest the app was working real smooth

Reply:
Probably because of screenshot test. There the test check if the changes in UI are over a cartain percentage. The question is why the ui changed.

Maybe when we use `MediaQuery.platformBrightnessOf(context) == Brightness.dark` instead of `context.watch<ThemeProvider>().darkMode` it doesn't change when changing the theme

Reply:
yeah the screenshot test was failing I tried to look into this and still scratching my head,I am kinda stuck guess I need some help here ;(

Reply:
Heyy yeah sure, sorry for the late reply. You can regenerate them by running `flutter test --update-goldens`.